package.json

```diff
@@ -8,7 +15,6 @@
     "gulp-eslint": "^2.0.0",
     "gulp-load-plugins": "^1.2.2",
     "proxy-middleware": "^0.15.0",
-    "url": "^0.11.0",
-    "web-component-tester-istanbul": "git://github.com/TimvdLippe/web-component-tester-istanbul.git#coverage-log-test-result-fix"
```

Comment:
`web-component-tester` is installed globally so `web-component-tester-istanbul` also needs to be installed globally to work. Once it is installed globally, there is no need for it to also be installed locally. That is why I removed `web-component-tester-istanbul` from the `devDependencies`.